spec/views/orders/index.html.erb_spec.rb

```diff
@@ -28,7 +28,8 @@
     ])
   end
 
-  it "renders a list of orders" do
```

Comment:
May I use such way to include js to my html? <script src="/assets/orders.js?body=1"></script>?
In this case I don't need to use yield :javascript or yield :head